docs/rules/no-firestore-object-arrays.md

```
@@ -3,3 +3,64 @@
 💼 This rule is enabled in the ✅ `recommended` config.
 
 <!-- end auto-generated rule header -->
+
+### What this rule enforces
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)